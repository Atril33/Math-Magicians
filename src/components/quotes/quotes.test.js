import renderer from 'react-test-renderer';
import MyQuotes from './quotes';

test("Snapshot testing of Quotes.js", () => {
 const comp = renderer.create(
   <MyQuotes />
 );
 let tree = comp.toJSON();
 expect(tree).toMatchSnapshot();
});