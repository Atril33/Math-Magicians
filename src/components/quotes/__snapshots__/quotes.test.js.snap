// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot testing of Quotes.js 1`] = `
<center>
  <div
    className="myQuote"
  >
    <p
      className="my-quote"
    />
    <p
      className="my-author"
    />
    <img
      alt="Loading..."
      className="my-image"
      src="loading.gif"
    />
  </div>
</center>
`;
